src/Cli/func/StaticResources/node-v4-templates.json

```diff
@@ -614,5 +614,47 @@
       "enabledInTryMode": false,
       "userPrompt": []
     }
+  },
+  {
+    "id": "McpToolTrigger-Typescript-4.x",
+    "runtime": "2",
+    "files": {
+      "%functionName%.ts": "import { app, InvocationContext, output } from \"@azure/functions\";\n\nconst PROPERTY_TYPE = \"string\";\n\n// Hello function - responds with hello message\nexport async function mcpToolHello(_toolArguments:unknown, context: InvocationContext): Promise<string> {\n    // Get name from the tool arguments\n    const mcptoolargs = context.triggerMetadata.mcptoolargs as {\n        name?: string;\n    };\n    const name = mcptoolargs?.name;\n\n    console.info(`Hello ${name}, I am MCP Tool!`);\n\n    return `Hello ${name}, I am MCP Tool!`;\n}\n\n// Register the hello tool\napp.mcpTool('hello', {\n    toolName: 'hello',\n    description: 'Simple hello world MCP Tool that responses with a hello message.',\n    toolProperties:[\n    {\n      propertyName: \"name\",\n      propertyType: PROPERTY_TYPE,\n      description: \"Required property to identify the caller.\",\n      isRequired: true,\n    }],\n    handler: mcpToolHello\n});"
```

Comment:
Template seems to be OK, but it looks like we're still pulling the old ext bundle as it does not find the binding - is the new ext bundle with mcp fully released?

```
[2025-10-24T00:15:31.369Z] The 'hello' function is in error: The binding type(s) 'mcpToolTrigger' were not found in the configured extension bundle. Please ensure the type is correct and the correct version of extension bundle is configured.
```